rest-api/etl/etl.sql

```diff
@@ -891,7 +948,7 @@ SELECT DISTINCT
     YEAR(b.date_of_birth)                       AS year_of_birth,
     MONTH(b.date_of_birth)                      AS month_of_birth,
     DAY(b.date_of_birth)                        AS day_of_birth,
-    TIMESTAMP(b.date_of_birth)                  AS birth_datetime,
+    NULL                                        AS birth_datetime,
```

Comment:
Why is this NULL?

Reply:
In the source data there is no birth or procedure time information. But both cases are fine. If you need datetime field, we can change it back.

Reply:
I don't know -- it all depends on how this field gets used. My assumption is that some analyses will expect this field to be set and won't bother checking date_of_birth if it isn't?

Reply:
After your question, we have asked about rules for populating *_datetime fields when time is unknown, CDM builder team says that it is fine to populate *_datetime fields when there is no time in the source data. So, we can populate it again.
http://forums.ohdsi.org/t/should--datetime-fields-be-required/3686/7